tests/test_create_sentence.rs

```diff
@@ -39,6 +39,82 @@ fn test_post_sentence_returns_200() {
     );
 }
 
+#[test]
+fn test_post_sentence_without_structure_returns_200() {
+
+    let connection = db::get_connection();
+    db::clear(&connection);
+
+    let mut json = HashMap::new();
+    let sentence_uuid = uuid::Uuid::new_v4();
+    json.insert("id", sentence_uuid.to_string());
+    json.insert("text", "This is a sentence.".to_string());
+    json.insert("iso639_3", "eng".to_string());
+
+    let client = reqwest::Client::new();
+
+    let url = format!(
+        "{}/sentences",
+        tests_commons::SERVICE_URL,
+    );
+    let response = client.post(&url)
+        .json(&json)
+        .send()
+        .unwrap();
+
+    assert_eq!(
+        response.status(),
+        StatusCode::Created,
+    );
+
+    assert_eq!(
+        db::get_structure(
+            &connection,
+            &sentence_uuid,
+        ),
+        "".to_string(),
```

Comment:
should be null in database

Reply:
Does it make sense to return an Option<T> here from `get_structure` ?